examples/tutorials/tutorial_dynamic_derivatives.py

```diff
@@ -0,0 +1,143 @@
+                fl.SurfaceEdgeRefinement(
+                    name="trailingEdge",
+                    method=fl.HeightBasedRefinement(value=0.001),
+                    edges=geometry["trailingEdge"],
+                ),
+            ],
+        ),
+        reference_geometry=fl.ReferenceGeometry(
+            moment_center=[0, 0, 0],
+            moment_length=[1, 1, 1],
+            area=2,
+        ),
+        operating_condition=fl.AerospaceCondition.from_mach(
+            mach=0.1469317682550648,
+            alpha=0 * fl.u.deg,
+            beta=0 * fl.u.deg,
+            thermal_state=fl.ThermalState(
+                temperature=288.15,
+                material=fl.Air(
+                    dynamic_viscosity=fl.Sutherland(
+                        reference_temperature=288.15,
+                        reference_viscosity=4.2925193198151646e-8 * fl.u.flow360_viscosity_unit,
+                        effective_temperature=110.4,
+                    )
+                ),
+            ),
+        ),
+        time_stepping=fl.Steady(
+            max_steps=10000, CFL=fl.RampCFL(initial=1, final=100, ramp_steps=1000)
+        ),
+        outputs=[
+            fl.VolumeOutput(
+                name="VolumeOutput",
+                output_fields=[
+                    "Mach",
+                ],
+            ),
+            fl.SurfaceOutput(
+                name="SurfaceOutput",
+                surfaces=geometry["*"],
+                output_fields=[
+                    "Cp",
+                    "CfVec",
+                ],
+            ),
+        ],
+        models=[
+            fl.Rotation(
+                volumes=cylinder,
+                spec=fl.AngularVelocity(0 * fl.u.rad / fl.u.s),
+            ),
+            fl.Freestream(surfaces=farfield.farfield, name="Freestream"),
+            fl.Wall(surfaces=geometry["wing"], name="NoSlipWall"),
+            fl.Fluid(
+                navier_stokes_solver=fl.NavierStokesSolver(
+                    absolute_tolerance=1e-9,
+                    linear_solver=fl.LinearSolver(max_iterations=35),
+                ),
+                turbulence_model_solver=fl.SpalartAllmaras(
+                    absolute_tolerance=1e-8,
+                    linear_solver=fl.LinearSolver(max_iterations=25),
+                ),
+            ),
+        ],
+    )
+
+# Run steady case with a fixed sliding interface for initializing the flow field.
+project.run_case(
+    params=params,
+    name="Tutorial Calculating Dynamic Derivatives using Sliding Interfaces (Steady)",
+)
+parent_case = fl.Case.from_cloud(project.case.id)
+
+# Update the parameters for the unsteady case.
+with fl.SI_unit_system:
+    params.time_stepping = fl.Unsteady(
+        max_pseudo_steps=80,
+        steps=400,
+        step_size=0.01 * 2.0 * math.pi / 20.0 * fl.u.s,
+        CFL=fl.RampCFL(initial=1, final=1e8, ramp_steps=20),
+    )
+    params.models[0].spec = fl.AngleExpression("0.0349066 * sin(0.05877271 * t)")
```

Comment:
Can you compute the 0.0349066 in expression ad-hoc
or
in documentation explain how the 0.034... is computed?